toolchain/musl/include/bits/wordsize.h

```diff
@@ -1 +0,0 @@
-#include <sys/user.h>
```

Comment:
Could you please split this into two commit, one removing these extra headers and one to update musl to 1.2.0.

Reply:
Done.